include/libdnf/advisory/advisory_set.hpp

```diff
@@ -41,11 +41,10 @@ namespace libdnf::advisory {
 class AdvisorySet {
 public:
     // TODO(dmach): move to common/exception.hpp, share across multiple sets/queries
-    struct UsedDifferentSack : public RuntimeError {
-        using RuntimeError::RuntimeError;
+    struct UsedDifferentSack : public Error {
+        UsedDifferentSack();
```

Comment:
This should be `libdnf_assert_same_base`, shouldn't it?

Reply:
OK. Make sense.